samples/bluetooth/fast_pair/locator_tag/src/main.c

```diff
@@ -301,15 +331,14 @@ static void fmdn_clock_synced(void)
 		 * This lets the Seeker detect the device and synchronize the clock even
 		 * if a significant clock drift occurred.
 		 */
-		app_fp_adv_mode_set(APP_FP_ADV_MODE_OFF);
+		app_fp_adv_request(&fp_adv_trigger_clock_sync, false);
 	}
 }
 
 static void fmdn_provisioning_state_changed(bool provisioned)
 {
-	static bool is_first_state_changed_cb = true;
-
-	enum app_fp_adv_mode fp_adv_mode;
+	/* Used to determine if the clock sync is required. */
+	static bool first_state_change_after_bootup = true;
```

Comment:
consider defining a function

```
static bool fmdn_provisioning_state_changed_first_cb_after_bootup(void)
{
	static bool first_state_change_after_bootup = true;
	bool is_first_state_change_after_bootup = first_state_change_after_bootup;
	
	first_state_change_after_bootup = false;

	return is_first_state_change_after_bootup;
}
```

Reply:
I am worried that this may make things even more complicated here (looking only at the function name I have no idea what is the purpose of calling it). Variable used to be was far more intuitive.

Reply:
It simplifies by hiding the assignment management in the function, it is fine solution.
will rename function to better indicate what it is used for